client/src/main/java/tds/student/Student.java

```diff
@@ -0,0 +1,67 @@
+package tds.student;
+
+/**
+ * Represents a student taking an exam
+ */
+public class Student {
+    private long id;
+    private String studentId;
+    private String stateCode;
+    private String clientName;
+
+    /**
+     * @return numeric id for the student
+     */
+    public long getId() {
+        return id;
+    }
+
+    /**
+     * @param id numeric id for the student
+     */
+    public void setId(long id) {
+        this.id = id;
+    }
+
+    /**
+     * @return the external SSID in the ART system
+     */
+    public String getStudentId() {
```

Comment:
Yeah our default standard is to create a SSID, like FOO, and then the Alternate SSID gets an A prepended to it, so AFOO.  Again this is just what we do when creating our own users, these values will not be related in practice and have to do with the schools Student management systems I think.  

Anyways, it seems like maybe it doesn't matter once it gets to the TDS database side of things.  So the SSID (whichever is used) is sent to an ART rest endpoint to get the "student package" as part of the login.  Then after that it looks like they insert into r_studentkeyid whichever is used.  So it might not be relevant for your code, but maybe keep it in mind.